### PR TITLE
boards/openlabs-kw41z-mini: documentation fixes

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

This fixes the rendering of the `boards/openlabs-kw41z-mini` documentation and adds a note about the `boards/openlabs-kw41z-mini-256kib`

### Testing procedure

Run `make doc` and check the generated file at `doc/doxygen/html/group__boards__openlabs-kw41z-mini.html`

### Issues/PRs references
https://github.com/RIOT-OS/RIOT/pull/13524#issuecomment-597664359

### CODE REVIEW

File: boards/openlabs-kw41z-mini/doc.txt [hunk 1]
```diff
@@ -3,20 +3,23 @@
...
+instead of the `M41W9VT4` (512k ROM). The boards are otherwise identical.
+See @ref boards_openlabs-kw41z-mini-256kib if you happen to have this version.
 
 ### Programming Pinout
 
 Programming and debugging can be done with a Raspberry Pi (or equivalent)
 running OpenOCD.
 
-
+```
```
Comment (miri64) [inline boards/openlabs-kw41z-mini/doc.txt]: Either also just indent them or use `~~~~~~~~~` fencing. ``` fencing does not work with Doxygen markdown.

File: boards/openlabs-kw41z-mini/doc.txt [hunk 2]
```diff
@@ -30,33 +33,40 @@ running OpenOCD.
...
 
+    cd RIOT/examples/gnrc_networking
+    BOARD=openlabs-kw41z-mini CFLAGS+="-DKW41ZRF_ENABLE_LEDS=1" make -j4 flash
 
 ### Debug Uart Pinout
 
 It is also possible to use the Pi for connecting to the debug uart.
 
-
+```
```
Comment (miri64) [inline boards/openlabs-kw41z-mini/doc.txt]: Dito

File: boards/openlabs-kw41z-mini/doc.txt (hunk 1 above)
Comment (benpicco) [reply boards/openlabs-kw41z-mini/doc.txt]: It worked in my local version of `doxygen` (1.8.13)…
Will change it.

Comment (miri64) [reply boards/openlabs-kw41z-mini/doc.txt]: Maybe it was just with syntax highlighting, not sure. Let's stay on the save side (Doxygen-version-wise).

File: boards/openlabs-kw41z-mini/doc.txt [hunk 3]
```diff
@@ -3,20 +3,23 @@
...
+See @ref boards_openlabs-kw41z-mini-256kib if you happen to have this version.
 
 ### Programming Pinout
 
 Programming and debugging can be done with a Raspberry Pi (or equivalent)
 running OpenOCD.
 
-
+~~~~~~~~~
     Pi GPIO Pins       kw41z-mini
```
Comment (miri64) [inline boards/openlabs-kw41z-mini/doc.txt]: Not sure why, but in the rendered version `GPIO` now has a link to [here](https://doc.riot-os.org/cc26xx__cc13xx__gpio_8h.html#a1037b18e2d226fe7d327d4a6f17a21c1) on my machine (doxygen v1.8.13, so same used to render the online doc), maybe use indentation after all? With that I don't have the problem at all.

File: boards/openlabs-kw41z-mini/doc.txt [hunk 4]
```diff
@@ -3,20 +3,23 @@
 @ingroup     boards
 @brief       Support for openlabs-kw41z-mini
 
- ### General information
+### General information
 
 This is an open-source development board shipped by openlabs.co with
 source files located at https://openlabs.co/OSHW/kw41z-mini
 
 A driver for the radio transceiver is available in #12277 (802.15.4 only).
```
Comment (miri64) [inline boards/openlabs-kw41z-mini/doc.txt]: Maybe also add a link here
```suggestion
A driver for the radio transceiver is available in
[#12277](https://github.com/RIOT-OS/RIOT/pull/12277) (802.15.4 only).
```

File: boards/openlabs-kw41z-mini/doc.txt (hunk 1 above)
Comment (benpicco) [reply boards/openlabs-kw41z-mini/doc.txt]: `rg -F "\`\`\`"` in `boards/` shows quite a few results. 

e.g. [`common/slwstk6000b`](https://github.com/RIOT-OS/RIOT/blob/master/boards/common/slwstk6000b/doc.txt#L182) or [`cc1312-launchpad/doc.txt`](https://github.com/RIOT-OS/RIOT/blob/master/boards/cc1312-launchpad/doc.txt#L58)

File: boards/openlabs-kw41z-mini/doc.txt (hunk 3 above)
Comment (miri64) [reply boards/openlabs-kw41z-mini/doc.txt]: Same problem with ```. Only indentation seems to fix that.

File: boards/openlabs-kw41z-mini/doc.txt (hunk 1 above)
Comment (miri64) [reply boards/openlabs-kw41z-mini/doc.txt]: See https://github.com/RIOT-OS/RIOT/pull/13617#discussion_r391120201.

File: boards/openlabs-kw41z-mini/doc.txt (hunk 3 above)
Comment (benpicco) [reply boards/openlabs-kw41z-mini/doc.txt]: Alright, indented it.

**Pull request merged**